### PR TITLE
[native] Ensure inheritance of fbos version for proxygen

### PR DESCRIPTION
**Pull request by czentgr**
When the setup scripts from velox are sourced the
FB_OS_VERSION is inherited. Proxygen should use the version of the FBOS dependencies.

This also removes the need for additional changes
when the Velox submodule is updated.

## Contributor checklist

- [x] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [x] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [x] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [x] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [x] Adequate tests were added if applicable.
- [x] CI passed.

## Release Notes
Please follow [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) and fill in the release notes below.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by czentgr** [comment]
@majetideepak FYI

**Review by majetideepak — APPROVED** [review]
@czentgr  Thanks! Were you able to verify that the correct value is being inherited locally?

**Comment by czentgr** [comment]
> @czentgr Thanks! Were you able to verify that the correct value is being inherited locally?

Yes, I was able to verify. Initially, I wasn't so sure myself if sourcing would preserve the variable from the respective velox setup script. So I did the test prior to opening the PR.

**Comment by czentgr** [comment]
@majetideepak I've rebased to bring it up to date. Is it possible we can merge this or are we waiting for another review?

**Comment by majetideepak** [comment]
@czentgr This is good to merge. I approved it but missed merging it in the past. Thanks for the rebase. I will land after CI finishes.

**Pull request merged**
